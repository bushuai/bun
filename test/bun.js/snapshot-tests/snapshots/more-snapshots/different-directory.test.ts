test("snapshots in different directory", () => {
  expect(1).toMatchSnapshot();
});
