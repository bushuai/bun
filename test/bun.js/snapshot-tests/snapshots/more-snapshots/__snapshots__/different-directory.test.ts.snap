// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots in different directory 1`] = `1`;
